packages/core/src/docdb/commands/createCluster.ts

```diff
@@ -28,15 +29,15 @@ export async function createCluster(node?: DocumentDBNode) {
         }
 
         span.record({ awsRegion: node?.client.regionCode })
-        const wizard = new CreateClusterWizard(node?.regionCode ?? '', {}, node?.client)
+        const wizard = new CreateClusterWizard(node?.client, {})
         const result = await wizard.run()
 
         if (!result) {
             getLogger().info('CreateCluster canceled')
             throw new ToolkitError('User cancelled wizard', { cancelled: true })
         }
 
-        const clusterName = result.ClusterName
+        const clusterName = result.RegionalCluster.DBClusterIdentifier ?? result.ElasticCluster.clusterName
```

Comment:
Nit: I think I've seen this logic a couple of times. Should it be encapsulated as a computed property?